addons/necropost-highlighter/addon.json

```
@@ -0,0 +1,185 @@
+{
+  "name": "Necropost highlighter",
+  "description": "Spot old forum topics brought back to life, before opening the message. Topics started much earlier than others listed on the same page are highlighted (in selected forums). If you've wasted time reading zombie messages or a help/question/request that was actually asked years ago, this is for you!",
+  "info": [
+    {
+      "type": "notice",
+      "id": "whatGetsHighlighted",
+      "text": "Sometimes an older topic is kept alive for good reasons, like an ongoing conversation, or the original poster following up their own topic months later. All older topics in selected forums are highlighted, including occasional lingering topics that are not necroposts. (That's why they get highlighted, but not removed.)"
+    }
+  ],
+  "credits": [
+    {
+      "name": "RT_Borg",
+      "link": "https://github.com/RT-Borg"
+    }
+  ],
+  "dynamicDisable": true,
+  "dynamicEnable": true,
+  "enabledByDefault": false,
+  "tags": ["community", "forums"],
+  "versionAdded": "1.28.0",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["^https://scratch.mit.edu/discuss/(?:m/)?(?:search|1|3|4|6|7|9|11|49)/.*"]
```

We don't need wildcard at the end? Query params are stripped from the end of the URL
```suggestion
      "matches": ["^https://scratch.mit.edu/discuss/(?:m/|)(?:search|1|3|4|6|7|9|11|49)/"]
```